src/App.js

```diff
@@ -1,23 +1,20 @@
-import logo from './logo.svg';
 import './App.css';
+import Header from './components/header';
+import HomeButtons from './components/buttons';
+import ReviewsPage from './components/reviewsPage';
+import {BrowserRouter, Routes, Route} from "react-router-dom"
+
 
 function App() {
+
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Routes>
```

Comment:
If you want the Header on every route you can render it above the Routes component.
Individual Route components should be the aspect of the page subject to the conditional rendering based on the URL.